frontend/public/components/graphs/area.tsx

```diff
@@ -93,6 +94,7 @@ type AreaChartProps = {
   humanize?: Humanize;
   height?: number,
   loading?: boolean;
+  minHeight?: number;
```

Comment:
I don't see this value ever being used, in which case, do we need a prop for it?